.github/workflows/ci.yml

```diff
@@ -213,7 +213,7 @@ jobs:
         python -m pip install pytest
         python -m pip install pytest-xdist
         echo ::group::Test Logs
-        python -m pytest -n auto -sv --max-worker-restart 512 | tee test.log 2>&1
+        python -m pytest --xfail_strict -n auto -sv --max-worker-restart 512 | tee test.log 2>&1
```

Comment:
To get the relevant exit code we should remove the pipe and the next lines or preserve the exit code.

Reply:
The argument is not working

Reply:
I can put strict in the mark itself